### PR TITLE
fix: aligning labels #406, #402, #403, #426

### LINKED ISSUE #402: Radio button: Text a few pixels too high up
## Description

When compared to the designs the Label for radio buttons on mobile seems to be two pixels above where the designed label is. On desktop the text is 1 pixel above where it should be.

## Steps to reproduce

1. Go to http://opensource.corp.adobe.com/spectrum-css/
2. Click on ...
3. Observe ...

## Screenshots

###Mobile screenshot
image

###Desktop screenshot
image

## Environment
 - **Spectrum CSS version:** 
 - **Browser(s) and OS(s):** 
Found on https://reactspectrum.blob.core.windows.net/reactspectrum/20e5131cfc8f65620868f16dc32defaedd215aaf/index.html?path=/story/radio--default

### LINKED ISSUE #403: Switch label spacing slightly off
## Description

This is the same problem as on the [Radio button component label bug](https://github.com/adobe/spectrum-css/issues/402). On the the Label for switches on mobile it seems to be two pixels above where the designed label is. On desktop the text is 1 pixel above where it should be.

## Steps to reproduce

1. Go to http://opensource.corp.adobe.com/spectrum-css/
2. Click on ...
3. Observe ...

## Screenshots

Desktop 
image

Mobile
image

## Environment
 - **Spectrum CSS version:** 
 - **Browser(s) and OS(s):** 

https://reactspectrum.blob.core.windows.net/reactspectrum/20e5131cfc8f65620868f16dc32defaedd215aaf/index.html?path=/story/switch--default

### LINKED ISSUE #406: Checkbox label position 1px/2px too low
## Description
Checkbox label is 1px (desktop) / 2px (mobile) lower than the design

## Steps to reproduce

1. Go to https://opensource.adobe.com/spectrum-css/components/checkbox/
2. Screenshot checkbox
3. Compare with UI kit

## Expected behavior
Label should be 1px higher (desktop) or 2px higher (mobile)

## Screenshots
(red is design)
desktop:
checkbox-desktop

mobile:
checkbox-mobile

## Environment
 - **Spectrum CSS version:** 
 - **Browser(s) and OS(s):** 

## Additional context
Came across while doing React review. Not sure if this is some rendering issue or not :( sorry if this is not relevant!

### LINKED ISSUE #426: Status light text is 1px too low
Screen Shot 2019-12-08 at 9 23 02 AM

Status light text baseline text is 1px too low relative to the circle

### PR DESCRIPTION
**Pull request by bernhard-adobe**
Correcting the top margin for checkboxes & radio buttons.
https://jira.corp.adobe.com/browse/SDS-4420 closes #406,
https://jira.corp.adobe.com/browse/SDS-4418 closes #402,
https://jira.corp.adobe.com/browse/SDS-4403 closes #403
https://jira.corp.adobe.com/browse/SDS-4472 closes #426 

Corrects #412
same as https://github.com/adobe/react-spectrum-v3/pull/138

## How and where has this been tested?
Inspected pages: 
http://localhost:3000/docs/radio.html
http://localhost:3000/docs/checkbox.html
http://localhost:3000/docs/toggle.html
http://localhost:3000/docs/switch.html
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## Screenshots

Screen Shot 2019-12-19 at 13 59 02
Screen Shot 2019-12-19 at 13 56 37
Screen Shot 2019-12-19 at 13 56 29
Screen Shot 2019-12-19 at 13 56 20

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: components/statuslight/index.css [hunk 1]
```diff
@@ -13,7 +13,9 @@ governing permissions and limitations under the License.
 @import '../commons/index.css';
 
 :root {
-  --spectrum-statuslight-padding-y: var(--spectrum-global-dimension-size-75);
+  /* Hardcoded for wrapping study.
+  Will be a DNA token in https://jira.corp.adobe.com/browse/SDS-4466 */
+  --spectrum-statuslight-padding-y: var(--spectrum-global-dimension-size-65);
```
Comment (lazd) [inline components/statuslight/index.css]: This was already fixed and verified in #426. What changed? I see no additional comments on that issue.

Comment (jianliao) [reply components/statuslight/index.css]: @lazd @bernhard-adobe I ran a VR test to help you see the difference. https://spectrum-visual-regression.ci.corp.adobe.com/job/spectrum-css-vr-test-asset/139/artifact/spectrum-css/backstop_data/html_report/index.html

Comment (jianliao) [reply components/statuslight/index.css]: A VR test like only takes less than 10mins on jenkins. I encourage you all run at least one to see the change for this kind of PR.

Comment (lazd) [reply components/statuslight/index.css]: Thanks, @jianliao. I'm wondering why this needed adjustment after a verified fix was already in.

**Comment by lazd** [comment]
@bernhard-adobe wondering why this change was needed after we already verified #426? What changed?

**Comment by lazd** [comment]
Talked to @bernhard-adobe about this, turns out the original fix was not verified properly.

Tested this for Checkbox and it looked good to go, so I'll go ahead and merge. Thanks!

**Comment by bernhard-adobe** [comment]
Thanks @lazd

**Comment by bernhard-adobe** [comment]
As requested by Spectrum-React here another comparison.
Browsers left to right:

1. Google Chrome Version 81.0.4017.0 (Official Build) canary (64-bit)on MacOS
2. Firefox 71.0 (64-bit) on MacOS
3. Safari Version 12.1.1 (14607.2.6.1.1) on MacOS
4. MS Edge Version 79.0.309.58 (Official build) Beta (64-bit) on MacOS
5. MS Edge Microsoft Edge 42.17134.1098.0 on Windows 10

There is a stronger sub-pixel difference between Firefox and all the other browsers with the correct non-baseline alignment. The line-height fix still provides the most consistent experience across browsers. 

Screenshot (might want to open in a new tab and zoom in a bit):
all-browsers

Difference between Safari & Chrome (and the other browsers):
firefox 1px off

**Pull request merged**